Dockerfile

```diff
@@ -9,21 +9,31 @@ RUN if [ -z "$PUBLISH_DIR" ]; then \
     echo "ERROR: PUBLISH_DIR build argument is required" && exit 1; \
     fi
 
+# Add build argument for executable name
+ARG EXECUTABLE_NAME
+
+# Error out if EXECUTABLE_NAME is not set
```

Comment:
We want to run in a distroless environment, consequently steps like these will not work.  Is it possible to do the validation before building the dockerfile?